packages/ember-htmlbars/lib/compat/helper.js

```diff
@@ -97,7 +97,7 @@ function HandlebarsCompatibleHelper(fn) {
     fnResult = fn.apply(this, args);
 
     if (options.element) {
-      Ember.deprecate('Returning a string of attributes from a helper inside an element is deprecated.');
+      Ember.deprecate('Returning a string of attributes from a helper inside an element is deprecated.', false, { id: 'ember-htmlbars.handlebars-compat-helper', until: '3.0.0' });
```

Comment:
Let's set this to 2.0.0, and add an issue to remove with a [CLEANUP beta] PR soon.